docs/src/pages/vue-components/select.md

```diff
@@ -140,6 +140,12 @@ You can dynamically load new options when scroll reaches the end:
 
 ## Filtering and autocomplete
 
+### Understanding the difference
+
+The intended difference between a filtering QSelect and an autocomplete QSelect is offering the user a set of options that could be the full intended text to be entered (autocomplete), or removing options in the selection according to the input by the user (filter).
```

Comment:
I think this is still confusing. Here's my suggestion:

`The intended difference between a filtering QSelect and an autocomplete QSelect is that autocomplete allows you  the model to accept a value that is not in the list of options, without adding it to the list. Filter on the other hand will result in a null or default value for the model if an option is not selected.`

Reply:
I think that would be good the technical part below. The first paragraph is what the user should expect, The second the developer.

Scott